app/frontend/.gitignore

```diff
@@ -23,9 +23,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-src/pb/**
```

Comment:
Don't we still need `src/proto/**` instead to stop people accidentally commit the generated js files?

Reply:
We need to include the folder, because `protoc` won't generate it now; or we could add it to the top of the script. (But the python side needs a script in that folder, so for consistency I'd prefer to have them all)